spec/integration/user_provides_new_ldap_pw_spec.rb

```diff
@@ -123,39 +112,6 @@
       expect(flash[:error]).to match(/Your OLD password was wrong/)
     end
 
-    it 'provides new ldap password and entered wrong new password' do
```

Comment:
this functionality should still be available
just the forgot password functionality is being dropped